plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/BaseOracleConnectorTest.java

```diff
@@ -282,6 +281,21 @@ public void testShowCreateTable()
                         ")");
     }
 
+    @Override
+    public void testAggregationWithUnsupportedResultType()
```

Comment:
Why override?

Reply:
ProjectNode is added above TableScan for HyperLogLog query.

Reply:
oh, because there is a coercion, and this is because we don't support `bigint` in oracle ?

Reply:
We can write `bigints` as `number(19)` but reads of `number(19)` happen as a `decimal`.

Reply:
let's have some code comment indication why override